doc/source/cluster/kubernetes/k8s-ecosystem/volcano.md

```diff
@@ -0,0 +1,319 @@
+#     memory: 4Gi
+#   queue: kuberay-test-queue
+# status:
+#   conditions:
+#   - lastTransitionTime: "2022-12-01T04:43:31Z"
+#     reason: tasks in gang are ready to be scheduled
+#     status: "True"
+#     transitionID: f89f3062-ebd7-486b-8763-18ccdba1d585
+#     type: Scheduled
+#   phase: Running
+```
+
+And checking the status of our queue to see that we have 1 running job:
+
+```shell
+kubectl get queue kuberay-test-queue -o yaml
+
+# apiVersion: scheduling.volcano.sh/v1beta1
+# kind: Queue
+# metadata:
+#   creationTimestamp: "2022-12-01T04:43:21Z"
+#   generation: 1
+#   name: kuberay-test-queue
+#   resourceVersion: "4427348"
+#   uid: a6c4f9df-d58c-4da8-8a58-e01c93eca45a
+# spec:
+#   capability:
+#     cpu: 4
+#     memory: 6Gi
+#   reclaimable: true
+#   weight: 1
+# status:
+#   reservation: {}
+#   running: 1
+#   state: Open
+```
+
+Next, we'll add an additional RayCluster with the same configuration of head / worker nodes, but a different name:
+
+```shell
+# Path: kuberay/ray-operator/config/samples
+# Includes labels `ray.io/scheduler-name: volcano` and `volcano.sh/queue-name: kuberay-test-queue` in the metadata.labels
+# Replaces the name to test-cluster-1
+sed 's/test-cluster-0/test-cluster-1/' ray-cluster.volcano-scheduler-queue.yaml | kubectl apply -f-
+```
+
+Now check the status of its PodGroup to see that its phase is `Pending` and the last status is `Unschedulable`:
+
+```shell
+kubectl get podgroup ray-test-cluster-1-pg -o yaml
+
+# apiVersion: scheduling.volcano.sh/v1beta1
+# kind: PodGroup
+# metadata:
+#   creationTimestamp: "2022-12-01T04:48:18Z"
+#   generation: 2
+#   name: ray-test-cluster-1-pg
+#   namespace: test
+#   ownerReferences:
+#   - apiVersion: ray.io/v1alpha1
+#     blockOwnerDeletion: true
+#     controller: true
+#     kind: RayCluster
+#     name: test-cluster-1
+#     uid: b3cf83dc-ef3a-4bb1-9c42-7d2a39c53358
+#   resourceVersion: "4427976"
+#   uid: 9087dd08-8f48-4592-a62e-21e9345b0872
+# spec:
+#   minMember: 3
+#   minResources:
+#     cpu: "3"
+#     memory: 4Gi
+#   queue: kuberay-test-queue
+# status:
+#   conditions:
+#   - lastTransitionTime: "2022-12-01T04:48:19Z"
+#     message: '3/3 tasks in gang unschedulable: pod group is not ready, 3 Pending,
+#       3 minAvailable; Pending: 3 Undetermined'
+#     reason: NotEnoughResources
+#     status: "True"
+#     transitionID: 3956b64f-fc52-4779-831e-d379648eecfc
+#     type: Unschedulable
+#   phase: Pending
+```
+
+Because our new cluster requires more CPU and RAM than our queue will allow, even though we could fit one of the pods with the remaining 1 CPU and 2Gi of RAM, none of the cluster's pods will be placed until there is enough room for all the pods. Without using Volcano for gang scheduling in this way, one of the pods would ordinarily be placed, leading to the cluster being partially allocated, and some jobs (like [Horovod](https://github.com/horovod/horovod) training) getting stuck waiting for resources to become available.
```

Comment:
```suggestion
Because the new cluster requires more CPU and RAM than our queue allows, even though one of the pods would fit in the remaining 1 CPU and 2Gi of RAM, none of the cluster's pods are placed until there is enough room for all the pods. Without using Volcano for gang scheduling in this way, one of the pods would ordinarily be placed, leading to the cluster being partially allocated, and some jobs (like [Horovod](https://github.com/horovod/horovod) training) being stuck waiting for resources to become available.
```